lua/astrocommunity/test/neotest/init.lua

```diff
@@ -5,6 +5,13 @@ return {
     "nvim-neotest/neotest-go",
     "nvim-neotest/neotest-python",
     "rouge8/neotest-rust",
+    {
+      "folke/neodev.nvim",
+      opts = function(_, opts)
+        require("astronvim.utils").list_insert_unique(opts.library.plugins, "neotest")
```

Comment:
```suggestion
        opts.library.plugins = require("astronvim.utils").list_insert_unique(opts.library.plugins, "neotest")
```

might want to use the "new" syntax for that even though the old one still works